tests/ha_master/run.sh

```diff
@@ -58,14 +58,8 @@ function test_evict_leader() {
 		if [ $i = 4 ]; then
 			continue
 		fi
-		# Leader evict is not effective immediately, we need to wait for a proper period of time.
-		for _ in {0..30}; do
-			NEW_LEADER_NAME=$(get_leader "$WORK_DIR" 127.0.0.1:${MASTER_PORT1})
-			if [ "$NEW_LEADER_NAME" != "$LEADER_NAME" ]; then
-				break
-			fi
-			sleep 1
-		done
+		NEW_LEADER_NAME=$(get_leader $WORK_DIR 127.0.0.1:${MASTER_PORT1})
```

Comment:
revert #1946

Reply:
https://github.com/pingcap/dm/pull/1946 shouldn't be reverted? What it focus on is different. The evict request is asynchronous, so new leader may not be elected immediately after the request returns.